R/page.R

```diff
@@ -28,12 +28,18 @@ page <- function(..., title = NULL, theme = bs_theme(), lang = NULL) {
       lang = lang,
       # Components require Bootstrap 5+
       if (isTRUE(theme_version(theme) >= 5)) component_dependencies(),
-      if (is_installed("shiny", "1.8.1.9001")) shiny::useBusyIndicators()
+      use_busy_indicators()
     ),
     theme = theme
   )
 }
 
+use_busy_indicators <- function() {
+  # Use busy indicators were added in shiny 1.8.1.9001 (after 1.8.1.1)
+  ubi <- asNamespace("shiny")[["useBusyIndicators"]]
```

Comment:
Any reason not to use `$`?

```suggestion
  ubi <- asNamespace("shiny")$useBusyIndicators
```

Reply:
Partial matching? It's unlikely we'll have another function that starts with `useBusyIndicators` -- `useBusyIndicatorsForFunAndProfit()`? -- but I tend to use `[[` as a strong signal that we know exactly what we're looking for